src/main/java/com/github/dadrus/App.java

```diff
@@ -11,7 +11,7 @@ public static void main( String[] args )
         System.out.println( "Hello World!" );
     }
 	
-	public void test() {
+	public void test() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)